src/views/developers/l10n.json

```diff
@@ -48,5 +48,5 @@
   "developers.faqDifferencesTitle": "What’s the difference between Blockly and Scratch Blocks?",
   "developers.faqDifferencesBody": "Scratch Blocks builds upon the Blockly code base, and is specifically designed with our principles in mind to support creative learning experiences.",
   "developers.faqCollabTitle": "I’d like to collaborate. How do I get in touch?",
-  "developers.faqCollabBody": "You can reach us over on <a href=\"https://github.com/LLK/\">github</a> or you can send an email to <a href=\"mailto:help@scratch.mit.edu\">help@scratch.mit.edu.</a> We look forward to hearing from you!"
+  "developers.faqCollabBody": "You can reach us over on <a href=\"https://github.com/LLK/\">github</a> or you can send an email to <a href=\"mailto:help@scratch.mit.edu\">help@scratch.mit.edu</a>. We look forward to hearing from you!"
```

Comment:
@Kenny2github we're trying to move away from messages with HTML as the `FormattedHTMLMessage` function (in react-intl) is deprecated. As long as you're making changes to the string, it would be a good idea to convert it to use placeholders for links and revise the place where it's used to avoid the deprecated call. There are examples for [removing FormattedHTMLMessage](https://github.com/LLK/scratch-www/wiki/Localization-Guide#removing-formattedhtmlmessage) in the wiki.

Reply:
Potentially done, let's see what Travis says